### PR TITLE
[TRA1][F12-B4]

### PR DESCRIPTION
**Pull request by bryanleejh**
As a group we tried to implement:
-Undo (Tu An)
-Change CSS Style (Bryan)
-Edit Command (Wang Yu)
-Commands Case Insensitive (He Yang)

We should have done this work on a separate branch instead of the main. Will be more careful next time.

### CODE REVIEW

File: src/seedu/addressbook/commands/EditCommand.java [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+            + "Edits the info of the person identified by the index number used in the last person listing.\n\t"
+            + "Parameters: INDEX [p]p/PHONE [p]e/EMAIL [p]a/ADDRESS  [t/TAG]...\n\t"
+            + "Example: " + COMMAND_WORD + " 1"
+            + " John Doe p/98765432 e/johnd@gmail.com a/311, Clementi Ave 2, #02-25 t/friends t/owesMoney";
+
+    public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Updated Person: %1$s";
+    public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book";
+
+
+    public EditCommand(int targetVisibleIndex, int itemNum, String newInfo, boolean isPrivate) throws IllegalValueException {
```
Comment (okkhoy) [inline src/seedu/addressbook/commands/EditCommand.java:38]: i think you can improve the SLAP on this one?

File: src/seedu/addressbook/parser/Parser.java [hunk 2]
```diff
@@ -16,16 +16,45 @@
...
-    public static final Pattern PERSON_DATA_ARGS_FORMAT = // '/' forward slashes are reserved for delimiter prefixes
-            Pattern.compile("(?<name>[^/]+)"
-                    + " (?<isPhonePrivate>p?)p/(?<phone>[^/]+)"
-                    + " (?<isEmailPrivate>p?)e/(?<email>[^/]+)"
-                    + " (?<isAddressPrivate>p?)a/(?<address>[^/]+)"
-                    + "(?<tagArguments>(?: t/[^/]+)*)"); // variable number of tags
-
+    public static final Pattern KEYWORDS_ARGS_FORMAT = Pattern.compile("(?<keywords>\\S+(?:\\s+\\S+)*)"); // one
+                                                                                                          // or
+                                                                                                          // more
```
Comment (okkhoy) [inline src/seedu/addressbook/parser/Parser.java:21]: haha! this happens when you do <kbd> Ctrl </kbd> + <kbd> Shift </kbd> + <kbd> F </kbd> without putting the comment on the new line :-P

**Review by okkhoy — COMMENTED** [review]
Good to see all of you have contributed to this exercise :+1:  please keep up the good team work :-)

**Comment by okkhoy** [comment]
@bryanleejh @WangYu-g @zhypaul @arishuynhvan 
Good job on this one!
As noted, you can be careful about branching when you do your project work.

Some comments added. Please ack & close the PR after reading comments.

File: src/seedu/addressbook/parser/Parser.java (hunk 2 above)
Comment (bryanleejh) [inline src/seedu/addressbook/parser/Parser.java]: Ahh okay will take a look at this again.

**Comment by bryanleejh** [comment]
Gonna double check why the checks are failing before moving on (: Thanks Akshay!

File: src/seedu/addressbook/parser/Parser.java (hunk 2 above)
Comment (bryanleejh) [reply src/seedu/addressbook/parser/Parser.java:21]: Oh yea :/ Sorry about this! This is so sloppy. Won't happen again.

File: src/seedu/addressbook/commands/EditCommand.java (hunk 1 above)
Comment (bryanleejh) [reply src/seedu/addressbook/commands/EditCommand.java:38]: Okay will take note!

**Pull request closed without merging**